src/style.css

```diff
@@ -1 +1,63 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+.search-bars {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
```

Comment:
You are fixing the container size by setting `width: 100%`. Let the content size dictate it to avoid overflow or accidental scroll bar unless there is such a requirement.